scaloid-common/src/main/st/org/scaloid/common/app.scala

```diff
@@ -96,7 +108,17 @@ trait SActivity extends Activity with SContext with TraitActivity[SActivity] wit
 
   protected override def onCreate(b: Bundle) {
     super.onCreate(b)
-    onCreateBodies.foreach(_ ())
+    AppHelpers.createBundle.withValue(Option(b)) {
+      onCreateBodies.foreach(_ ())
+    }
+  }
+
+  def onCreate(body: Option[Bundle] => Any) = {
+    val el = { () =>
+      body(AppHelpers.createBundle.value)
```

Comment:
No problem. I'm leaving the line comments here to help reduce future confusion. The Android (OO overridden) `onCreate` vs. the Scaloid (functional) `onCreate` might not be intuitive to all users at first glance.